api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -65,12 +64,18 @@ message PluginConfig {
   // Filter/service configuration used to configure or reconfigure a plugin
   // (proxy_on_configuration).
   google.protobuf.Any configuration = 4;
+
+  // If there is a fatal error on the VM (e.g. exception, abort(), on_start or on_configure return false),
+  // then all plugins associated with the VM will either fail closed (by default), e.g. by returning an HTTP 503 error,
+  // or fail open (if 'fail_open' is set to true) by bypassing the filter. Note: when on_start or on_configure return false
+  // during xDS updates the xDS configuration will be rejected and when on_start or on_configuration return false on initial
+  // startup the proxy will not start.
+  bool fail_open = 5;
```

Comment:
Wasn't the consensus that we should always blackhole the traffic?

Reply:
I was under the impression that Mandar still wanted a fail closed/fail open option for telemetry.  This only effects non-startup non-xDS cases BTW.

Reply:
If the telemetry extension fails to load, we should not blackhole traffic.
fail_open with default set to false is correct.